indexer/packages/postgres/src/models/candle-model.ts

```diff
@@ -77,4 +77,8 @@ export default class CandleModel extends Model {
   trades!: number;
 
   startingOpenInterest!: string;
+
+  orderbookMidPriceOpen: string | undefined;
```

Comment:
orderbookMidPriceOpen?: string;

Reply:
I think having the property be required but allowing the type to be undefined is easier to understand and safer in code.